posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,90 @@
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021 - OpenLiberty.io
+seo-description: A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty.
+blog_description: "A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile, Jakarta EE and Open Liberty at JCON-Online 2021
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Despite the global pandemic and general chaos of this year, many amazing developer events and conferences have battled through are still taking place, bring the developer community together once more - many virtually! JCON-Online is one such event. It's aiming to really utilise this digital format to bring together the international Java community. The great part is, it's free for all JUG members!
+
+Several of the Open Liberty team will be joining https://jcon.one/[JCON-Online 2021] on 5-8 October. If you're also going to be there, deciding which of the great talks to see at JCON looks like it could be a challenge. So we've put together a helpful summary of the sessions on https://microprofile.io/[MicroProfile], https://jakarta.ee/[Jakarta EE] and https://openliberty.io/about/[Open Liberty].
+
+
+== Talks at JCON 2021
+
+A selection of MicroProfile and Open Liberty talks on Tuesday, Wednesday and Thursday:
+
+*Tuesday 5th*
+* https://jcon.sched.com/event/jzzB/live-coding-configuring-your-cloud-native-applications-with-microprofile-config?iframe=no[Live coding: Configuring your cloud-native applications with MicroProfile Config]
++
+Java Champion https://twitter.com/emilyfhjiang[Emily Jiang] live demo session will showcase various ways to configure your microservices via the amazing features introduced in MicroProfile Config 2.0, including config profile, config properties, variable replacement and many more. 
+
+*Wednesday 6th*
+* https://jcon.sched.com/event/jzyu/exploring-stateful-microservices-built-with-open-liberty-in-kubernetes?iframe=no[Exploring Stateful Microservices built with Open Liberty in Kubernetes]
++
+CJUG's https://twitter.com/mgrygles[Mary Grygleski] will explore a tried and true, enterprise-grade, Java Open Source stack - Open Liberty in Kubernetes - which exercises and promises to fulfill the expectations of a modern server solution that epitomizes all the elements needed to build, run and deploy an efficient production system today. Mary will examine and demonstrate the fundamentals of such a system that’s built with Open Liberty in Kubernetes, including: Microservices/REST API, Concurrency, Stateful vs Stateless and Deployment.
```

Comment:
suggested edit:
Mary Grygleski] will explore a tried and true, enterprise-grade, Java Open Source stack - Open Liberty in Kubernetes - which exercises and promises to fulfill the expectations of a modern server solution that epitomizes all the elements needed to build, run and deploy an efficient production system today.
--->
Mary Grygleski] explores a tried and true, enterprise-grade, Java open source stack: Open Liberty in Kubernetes, a modern server solution that epitomizes all the elements needed to build, run and deploy an efficient production system today.